src/multi_vector_simulator/B0_data_input_json.py

```diff
@@ -245,7 +249,12 @@ def retrieve_date_time_info(simulation_settings):
 
 
 def load_json(
-    path_input_file, path_input_folder=None, path_output_folder=None, move_copy=False
+    path_input_file,
+    path_input_folder=None,
+    path_output_folder=None,
+    move_copy=False,
+    flag_missing_values=True,
+    set_default_values=False,
```

Comment:
Why is `set_default_values=False` here?

Reply:
Because we only want to set it to True in `cli.py` for the moment, so we do not have to look for uses of the `load_json` across many files